lib/pages/settings_views/epicbox_settings_view/epicbox_settings_view.dart

```diff
@@ -51,28 +54,28 @@ class _WalletEpicBoxSettingsViewState
             "Epic Box Server",
             style: STextStyles.titleH4(context),
           ),
-          // actions: [
-          //   Padding(
-          //     padding: const EdgeInsets.only(right: 8.0),
-          //     child: IconButton(
-          //       splashRadius: 20,
-          //       onPressed: () {
-          //         Navigator.of(context).pushNamed(
-          //           AddEditEpicBoxView.routeName,
-          //           arguments: const Tuple3(
-          //             AddEditEpicBoxViewType.add,
-          //             null,
-          //             EpicBoxSettingsView.routeName,
-          //           ),
-          //         );
-          //       },
-          //       icon: const PlusIcon(
-          //         width: 20,
-          //         height: 20,
-          //       ),
-          //     ),
-          //   ),
-          // ],
+          actions: [
```

Comment:
I think this section needs removing as we want to remove custom Epic Box server management for the time being